cli/command/container/diff_test.go

```diff
@@ -0,0 +1,95 @@
+package container
+
+import (
+	"context"
+	"io"
+	"strings"
+	"testing"
+
+	"github.com/docker/cli/internal/test"
+	"github.com/docker/docker/api/types/container"
+	"github.com/pkg/errors"
+	"gotest.tools/v3/assert"
+)
+
+func TestRunDiff(t *testing.T) {
+	cli := test.NewFakeCli(&fakeClient{
+		containerDiffFunc: func(
+			ctx context.Context,
+			containerID string,
+		) ([]container.FilesystemChange, error) {
+			return []container.FilesystemChange{
+				{
+					Kind: 0, // Modified ("C")
+					Path: "/path/to/file0",
+				},
+				{
+					Kind: 1, //  Added ("A")
+					Path: "/path/to/file1",
+				},
+				{
+					Kind: 2, // Deleted ("D")
+					Path: "/path/to/file2",
+				},
+			}, nil
+		},
+	})
+
+	cmd := NewDiffCommand(cli)
+	cmd.SetOut(io.Discard)
+
+	containerId := "a61d6b418006"
+	cmd.SetArgs([]string{containerId})
+
+	err := cmd.Execute()
```

Comment:
Same comment here as for the other test; it's probably fine to just inline this here (and use a value that describes what it is);

```go
cmd.SetArgs([]string{"container-id"})
err := cmd.Execute()
```